Commands/Tool.lua

```diff
@@ -99,14 +103,51 @@ end
 -----------------------------------------------
 -------------------SUPERPICK-------------------
 -----------------------------------------------
-function HandleSuperPickCommand(Split, Player)
-	if SP[Player:GetName()] == nil or not SP[Player:GetName()] then -- check if super pickaxe is activated
-		SP[Player:GetName()] = true
-		Player:SendMessage(cChatColor.LightPurple .. "Super pick activated")
-	else -- else deactivate the superpickaxe
-		SP[Player:GetName()] = false
-		Player:SendMessage(cChatColor.LightPurple .. "Super pick deactivated")
+function HandleSuperPickCommand(a_Split, a_Player)
+	-- //
+	-- /,
+	
+	-- A table containing all the ID's of the pickaxes
+	local Pickaxes = 
+	{
+		E_ITEM_WOODEN_PICKAXE,
+		E_ITEM_STONE_PICKAXE,
+		E_ITEM_IRON_PICKAXE,
+		E_ITEM_GOLD_PICKAXE,
+		E_ITEM_DIAMOND_PICKAXE,
+	}
```

Comment:
Is it the little performance gain worth? It's only created when the command is called after all. I'd rather have it here in the command handler. It makes it easier to read as well since you can directly see what `Pickaxes` is.